test/liveValidatorTests.ts

```diff
@@ -18,6 +18,7 @@ describe("Live Validator", () => {
     it("should initialize with defaults", () => {
       const options = {
         swaggerPaths: [],
+        excludedSwaggerPathsPattern: Constants.DefaultConfig.ExcludedSwaggerPathsPattern,
```

Comment:
Why do we need to add the default value assignment for each test?

Reply:
because the test is trying to verify the default options. Since we added the default excluded swagger patterns, we have to also add it in the test to match the behavior